deepchem/models/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,99 @@
+import torch
+
+try:
+  import pytorch_lightning as pl  # noqa
+  PYTORCH_LIGHTNING_IMPORT_FAILED = False
+except ImportError:
+  PYTORCH_LIGHTNING_IMPORT_FAILED = True
+
+
+class DCLightningModule(pl.LightningModule):
```

Comment:
Can you add this to the `docs/` folder so docs are generated for this new class?